receiver/awsecscontainermetricsreceiver/README.md

```diff
@@ -3,21 +3,173 @@
 ***Status:***
 This receiver is under development and not recommended for production usage.
 
-### Overview
+## Overview
 
-This receiver reads task metadata and docker stats from [Amazon ECS Task Metadata Endpoint](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint.html).
+AWS ECS Container Metrics receiver reads task metadata and docker stats from [Amazon ECS Task Metadata Endpoint](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint.html), and generates resource usage metrics from them. To get a full list of available metrics, see the `Available Metrics` section below.
 
-### Config
 
-An example config,
+## Configuration
+
+Example:
 
 ```yaml
+receivers:
   awsecscontainermetrics:
     collection_interval: 20s
 ```
 
 #### collection_interval
-
+```
 This receiver collects task metadata and container stats at a fixed interval and emits metrics to the next consumer of OpenTelemetry pipeline. `collection_interval` will determine the frequency at which metrics are collected and emitted by this receiver.
 
-default: `20s`
\ No newline at end of file
+default: `20s`
+```
+
+## Available Metrics
+Following is the full list of metrics emitted by this receiver.
+
+Task Level Metrics | Container Level Metrics | Unit 
+------------ | ------------- | --------------------
+ecs.task.memory.usage | container.memory.usage | Bytes
+ecs.task.memory.usage.max | container.memory.usage.max | Bytes
+ecs.task.memory.usage.limit | container.memory.usage.limit | Bytes
+ecs.task.memory.reserved | Bcontainer.memory.reserved | MB
+ecs.task.memory.utilized | container.memory.utilized | MB
+ecs.task.cpu.usage.total | container.cpu.usage.total | NS
+ecs.task.cpu.usage.kernelmode | container.cpu.usage.kernelmode | NS
+ecs.task.cpu.usage.usermode | container.cpu.usage.usermode | NS
+ecs.task.cpu.usage.system | container.cpu.usage.system | NS
```

Comment:
NS means nanoseconds?? 

I've never seen "NS" before. I have seen "ns" lower case though

Reply:
Used full form of units matching with CloudWatch unit conventions. https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_MetricDatum.html